experimental/auth-logic/wrappers/provenance_build_wrapper.go

```diff
@@ -70,8 +70,8 @@ func (pbw ProvenanceBuildWrapper) EmitStatement() (UnattributedStatement, error)
 
 	return UnattributedStatement{
 		Contents: fmt.Sprintf("\"%v::Binary\" hasProvenance(\"%v::Provenance\").\n",
-			applicationName, applicationName) +
+			sanitizedAppName, sanitizedAppName) +
```

Comment:
I thought this was more readable than a single `fmt.Sprintf` because these are separate statements. I did it a different way that is similar to what you suggested for the verification wrapper (using constants) and uses just one sprintf

Reply:
Then I think having each part as a variable with an expressive name, and a top-level `fmt.Sprintf` would be more readable.